ProjetINFRA/src/test/java/fr/m2dl/infra/OrchestratorTest.java

```diff
@@ -0,0 +1,27 @@
+package fr.m2dl.infra;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class OrchestratorTest {
+
+	@Test
+	public void createOrchestratorTest() {
+		Orchestrator orchestrator = new Orchestrator();
+		orchestrator.createAgent(new Agent() {
```

Comment:
Maybe we should create a seed generic Agent. That can be usefull because it will be used for other tests.